CHANGELOG.md

```diff
@@ -0,0 +1,381 @@
+- `getReceipt`/`verifyReceipt` RPCs (https://github.com/microsoft/CCF/pull/567)
+- Support for app-defined ACLs (https://github.com/microsoft/CCF/pull/590)
+
+Binaries for `cchost` and `libluagenericenc.so` are attached to this release. Note that libluagenericenc.so should be signed before being deployed by CCF (see https://microsoft.github.io/CCF/developers/build_app.html#standalone-signing).
+    
+## [0.5] - 2019-11-14
+
+This pre-release fixes minor issues and clarifies some of `cchost` command line options. 
+
+### Removed
+- The `new_user` function in constitution scripts (e.g. `gov.lua`) should be deleted as it is now directly implemented inside CCF (https://github.com/microsoft/CCF/pull/550). 
+- `cmake -DTARGET=all` replaced with `cmake -DTARGET=sgx;virtual`. See https://microsoft.github.io/CCF/quickstart/build.html#build-switches for new values (https://github.com/microsoft/CCF/pull/513). 
+
+### Changed
+- The members and users certificates can now be registered by the consortium using clients that are not the `memberclient` CLI (e.g. using the `tests/infra/jsonrpc.py` module) (https://github.com/microsoft/CCF/pull/550). 
+- Fix for Raft consensus to truncate the ledger whenever a rollback occurs and use `commit_idx` instead of `last_idx` in many places because of signatures (https://github.com/microsoft/CCF/pull/503). 
+- Join protocol over HTTP fix (https://github.com/microsoft/CCF/pull/550). 
+- Clearer error messages for when untrusted users/members issue transactions to CCF (https://github.com/microsoft/CCF/pull/530). 
+- `devcontainer.json` now points to right Dockerfile (https://github.com/microsoft/CCF/pull/543).
+- `cchost --raft-election-timeout` CLI option default now set to 5000 ms (https://github.com/microsoft/CCF/pull/559).
+- Better descriptions for `cchost` command line options (e.g. `--raft-election-timeout`) (https://github.com/microsoft/CCF/pull/559). 
+
+The `cchost`, `libluagenericenc.so`, `keygenerator` and `memberclient` are also attached to this release to start a CCF network with lua application. 
+Note that `libluagenericenc.so` should be signed before being deployed by CCF (see https://microsoft.github.io/CCF/developers/build_app.html#standalone-signing). 
+    
+## [0.4] - 2019-11-04
+
+In this preview release, it is possible to run CCF with the PBFT consensus algorithm, albeit with significant limitations.
+
+The evercrypt submodule has been removed, the code is instead imported, to make release tarballs easier to use.
+    
+## [0.3] - 2019-10-07
+
+This pre-release implements the genesis model described in the TR, with a distinct service opening phase. See https://microsoft.github.io/CCF/start_network.html for details.
+
+Some discrepancies with the TR remain, and are being tracked under https://github.com/microsoft/CCF/milestone/2
+
+## 0.2 - 2019-06-07
+
+Initial pre-release
+
+
+[Unreleased]: https://github.com/microsoft/CCF/compare/ccf-0.13.3...HEAD
```

Comment:
What does this do?

Reply:
Not sure what you mean. It's markdown, and the link shows the diff between the last version and HEAD: